jobs/ci-kubernetes-bazel-test.sh

```diff
@@ -25,6 +25,6 @@ bazel clean --expunge
 make bazel-test && rc=$? || rc=$?
 
 # Coalesce test results into one file for upload.
-"$(dirname "${0}")/../images/pull_kubernetes_bazel/coalesce.py"
+"$(dirname "${0}")/../images/bazelbuild/coalesce.py"
```

Comment:
I'm sure there is a reason we don't just put coalesce.py into the bazelbuild image, right?